### DISCUSSION TITLE
New x86_64 platform implementation?

### DISCUSSION THREAD

**Discussion by tiberio87** [pascaliske/infrastructure#1128] [Ideas]
Do you want to implement the x86_64 platform?

**Comment by pascaliske** [comment]
This repository is not tied to the ARM platform except some Ansible tasks which are only executed on ARM devices:

- [Setup of `log2ram` to avoid excessive wear on the SD card](https://github.com/pascaliske/infrastructure/blob/master/ansible/roles/common/tasks/tools/log2ram.yml)
- [Some Alpine 3.13 specific workarounds](https://github.com/pascaliske/infrastructure/blob/master/ansible/roles/common/tasks/packages.yml#L17-L40)
- [Configuring `cgroup` settings](https://github.com/pascaliske/infrastructure/blob/master/ansible/roles/common/tasks/system/boot.yml)
- [Expanding SD card filesystem](https://github.com/pascaliske/infrastructure/blob/master/ansible/roles/common/tasks/system/filesystem.yml)
- [Disabling swap for better IO performance with SD cards](https://github.com/pascaliske/infrastructure/blob/master/ansible/roles/common/tasks/system/swap.yml)

All of those tasks are meant for Raspberry Pi devices but are only bound to the [`ansible_architecture`](https://github.com/pascaliske/infrastructure/blob/master/ansible/group_vars/all.yml#L3) fact, which is enough in my case. Feel free to adapt it for your usage but I think it is not necessary to "implement" it in this repository - sorry. 🙂
